### ISSUE TITLE
Generate bionic images where possible

### ISSUE THREAD

**Issue by joseph-wakeling-sociomantic** [sociomantic-tsunami/cachalot#57]
While xenial-built packages should work on bionic, it would be convenient to be able to generate explicitly bionic (18.04) packages for some of our pre-built libraries (e.g. `libmxnet`).  This will need bionic images (at least for `sociomantictsunami/base`, as I imagine `dlang` will be problematic for the time being).

**Comment by leandro-lucarella-sociomantic** [comment]
We will just drop D1 support in Bionic, so no `tangort` and `dmd1` packages will be provided in Bionic.

**Comment by leandro-lucarella-sociomantic** [comment]
https://github.com/sociomantic-tsunami/cachalot/pull/59

**Comment by daniel-zullo** [comment]
It seems these issue has been addressed. Can I close it? Or I am missing something?

**Comment by leandro-lucarella-sociomantic** [comment]
Is true that support seems to be there, but bionic images are only being pushed for [develbase](https://cloud.docker.com/u/sociomantictsunami/repository/registry-1.docker.io/sociomantictsunami/develbase/tags), I guess we need to see why they are not being pushed for [runtimebase](https://cloud.docker.com/u/sociomantictsunami/repository/registry-1.docker.io/sociomantictsunami/runtimebase/tags) and [develdlang](https://cloud.docker.com/u/sociomantictsunami/repository/registry-1.docker.io/sociomantictsunami/develdlang/tags) and start pushing them before closing this.

**Comment by leandro-lucarella-sociomantic** [comment]
I guess if we remove the `IMAGES` [here](https://github.com/sociomantic-tsunami/cachalot/blob/v7.x.x/.travis.yml#L17) it should generate them and push them. But maybe it just fails too :)

**Comment by leandro-lucarella-sociomantic** [comment]
2a4769bea4381e6a608867bfb77a46b8f476a1fa explains why we are only building the base image. I'm not sure how many base tools are already being built for bionic, I guess if we try the build we'll find out :)

**Comment by leandro-lucarella-sociomantic** [comment]
`runtimebase` probably can be built for bionic too without issues, as it is even simpler than `develbase`, is just that the split was done after the introduction of bionic.

**Comment by leandro-lucarella-sociomantic** [comment]
https://github.com/sociomantic-tsunami/cachalot/pull/83

**Comment by leandro-lucarella-sociomantic** [comment]
It seems like now even `develbase` is failing. At least ebtree and libdrizzle-redux have no bionic builds:
```
Err:9 https://dl.bintray.com/sociomantic-tsunami/libdrizzle-redux bionic Release

  404  Not Found [IP: 52.35.217.225 443]

Err:10 https://dl.bintray.com/sociomantic-tsunami/mxnet bionic Release

  404  Not Found [IP: 52.35.217.225 443]
```
https://travis-ci.org/sociomantic-tsunami/cachalot/jobs/575744248

**Comment by daniel-zullo** [comment]
Thanks for investigating it and providing such a good report about the current status. I'll follow up on it  checking out the libraries that have not bionic builds.

**Comment by leandro-lucarella-sociomantic** [comment]
They are probably buildable in bionic without any issues, it's just that the packages need to be built by adding bionic to their CI and uploaded to the repos.

**Comment by daniel-zullo** [comment]
Adding Ubuntu bionic support for mxnet in sociomantic-tsunami/mxnet/pull/36

**Comment by daniel-zullo** [comment]
Adding Ubuntu bionic support for libdrizzle-redux sociomantic-tsunami/libdrizzle-redux/pull/341

**Comment by daniel-zullo** [comment]
The runtimebase image for bionic will be available once #84 gets released.

**Comment by daniel-zullo** [comment]
All images are now built for ubuntu xenial and bionic.
